.circleci/config.yml

```diff
@@ -51,7 +51,9 @@ jobs:
        - add_ssh_keys:
            fingerprints:
              - "fb:f3:fe:be:1c:b2:ec:b6:25:f9:7b:a6:87:54:02:8c"
+             - "9d:3b:fe:7c:09:3b:ce:a9:6a:de:de:41:fb:6b:52:62"
```

Comment:
this not going to work, you need two jobs, because the two keys have same host ("github.com") so circle will use the first one.